packages/shopify-codemod/transforms/global-reference-to-import.js

```diff
@@ -146,12 +155,17 @@ export default function globalReferenceToImport(
         });
 
       for (const anImport of imports.values()) {
-        const {file, name} = anImport;
+        const {file, name, hasNamedExports} = anImport;
         const {node: {body}} = path;
+        const importSpecifierBuilder = hasNamedExports ? (
```

Comment:
If you want to break this onto multiple lines (I don't know that it's necessary in this case anyways), preferred format would be

```js
const importSpecifierBuilder = hasNamedExports
  ? j.importNamespaceSpecifier
  : j.importDefaultSpecifier;
```

I would probably just do this check in the `j.importDeclaration` call instead, though, and store the built identifier so you don't have to repeat `j.identifier(name)`.